checkout-extension/src/Checkout.liquid

```diff
@@ -1,39 +1,124 @@
 {%- if flavor contains "react" -%}
 import {
+  reactExtension,
   Banner,
+  BlockStack,
+  Checkbox,
+  Text,
   useApi,
+  useApplyAttributeChange,
+  useInstructions,
   useTranslate,
-  reactExtension,
-} from '@shopify/ui-extensions-react/checkout';
+} from "@shopify/ui-extensions-react/checkout";
 
-export default reactExtension(
-  'purchase.checkout.block.render',
-  () => <Extension />,
-);
+// 1. Choose an extension target
+export default reactExtension("purchase.checkout.block.render", () => (
+  <Extension />
+));
 
 function Extension() {
   const translate = useTranslate();
   const { extension } = useApi();
+  const instructions = useInstructions();
+  const applyAttributeChange = useApplyAttributeChange();
+
 
+  // 2. Check instructions for feature availability
+  if (!instructions.attributes.canUpdateAttributes) {
+    // Consider rendering a fallback UI or nothing at all, if the feature is unavailable
+    return (
+      <Banner title="{{ name }}" status="warning">
+        Attribute changes are not supported in this checkout
```

Comment:
This could use `translate()`

Reply:
It could. My thinking was that you'd probably never actually output this to the buyer though and is more for demonstration purposes within the extension.

Reply:
I just thought that since it's a banner then it would add consistency. But, yeah, no biggie.